neqo-transport/tests/sim/taildrop.rs

```diff
@@ -71,6 +75,9 @@ impl TailDrop {
             dropped: 0,
             delivered: 0,
             maxq: 0,
+            start_time: None,
+            last_sent: None,
```

Comment:
Maybe call these first_packet_time and last_packet_time so it is clear what these are for.